examples/ubuntu-vm-hostpath-pv.yaml

```diff
@@ -0,0 +1,74 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: hostpath-dir-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  # Set volumeMode to Filesystem so the VM can consume this PVC
+  # as a Filesystem device
+  volumeMode: Filesystem
+  storageClassName: local-storage
+  resources:
+    requests:
+      storage: 100Mi
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: hostpath-dir-pv
+spec:
+  capacity:
+    storage: 100Mi
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: local-storage
+  # Set volumeMode to Filesystem so the VM can consume this PV
+  # as a Filesystem device
+  volumeMode: Filesystem
+  hostPath:
+    # Use a path that's visible inside the Virtlet pod.
+    # 9p mode must be supported in host kernel(command: modinfo 9p)
```

Comment:
Host kernel? Or should that be supported by guest kernel in VM image?

Reply:
great, you are right, guest kernel also must support 9p, the example use ubuntu os which support 9p. actually，I want to say that host kernel must install 9p mode. 
I shoud modify the annotation as "# 9p mode must be installed in host kernel(command: modinfo 9p)", is ok?

Reply:
Is it really needed on host? It's served by qemu which afaik do not uses any part of 9p from host kernel, it implements itself 9p protocol.

Reply:
OK, you are great again, I used "rmmod 9p" and test again, it works. I change the annotation "# 9p mode must be supported in guest kernel in VM image."